packages/shared/src/lib/log.ts

```diff
@@ -229,6 +229,11 @@ export enum LogEvent {
   ToggleClickbaitShield = 'toggle clickbait shield',
   ClickbaitShieldTitle = 'clickbait shield title',
   // End Clickbait Shield
+  // Start Share
+  ShareProfile = 'share profile',
+  ShareSource = 'share source',
+  ShareTag = 'share tag',
```

Comment:
why do we need these, did we not have it already?

Reply:
@capJavert Not these specific events as far as I could tell, no.

Added them cause @rebelchris mentioned all new events should be in this enum.

Reply:
ah I see, they were just not under the enum, but they are not new 👍